templates/Gemfile.erb

```diff
@@ -41,12 +41,19 @@ group :development, :test do
   gem "bullet"
   gem "bundler-audit", ">= 0.5.0", require: false
   gem "dotenv-rails"
-  gem "factory_girl_rails"
   gem "pry-byebug"
   gem "pry-rails"
   gem "rspec-rails", "~> 3.6"
 end
 
+group :development, :test, :staging do
```

Comment:
Looking at how Suspenders is set up out of the box, the Staging environment does not exist anymore. The Staging application runs under the Production environment. Is there a way you can think of that does not need us to include these gems in Production?

Reply:
Suspenders uses FactoryGirl by default for `dev:prime` (renamed to `data:prime` here).
We could skip that requirement... OR we could have a conditional install based on the existence of the ENV variable... yet again another reason to not do ENV level parity between staging and production :)